compiler/setup.py

```diff
@@ -252,12 +251,13 @@ def prepare_installation():
             "-GNinja",
             "--log-level=VERBOSE",
             "-DIREE_BUILD_PYTHON_BINDINGS=ON",
+            "-DIREE_BUILD_SAMPLES=OFF",
+            "-DIREE_BUILD_TESTS=OFF",
```

Comment:
@stellaraccident do you remember if we had a reason to include tests and samples in the compiler python release build? I wouldn't expect any of those code paths / executables / etc. to end up in the wheels anyways.

Reply:
I doubt I had a reason for not including these lines. Probably just forgot.